demo/src/main/res/layout/merge_player_controls.xml

```diff
@@ -23,20 +26,21 @@
 
   <SeekBar
     android:id="@+id/player_controls_progress"
-    style="@style/Controls.ScrubBar"
-    android:layout_width="0dp"
+    style="@style/Controls.SeekBar"
+    android:layout_width="@dimen/use_weight"
     android:layout_height="wrap_content"
     android:layout_weight="100"
     android:layout_gravity="center_vertical"
-    android:layout_marginLeft="12dp"
-    android:layout_marginRight="12dp" />
+    android:layout_marginLeft="@dimen/player_controls_seekbar_margin_horizontal"
```

Comment:
Why does the above `ImageView` require `marginStart` and `End` but this one doesn't?

Reply:
It's because they're the same, so it it redundant. You only need to specify `Start`/`End` when the two sides have asymmetrical margin/padding values (and Lint only flags those cases, accordingly) since if you mirror them, they're still the same :)